resources/js/bots/wallet/bot.js

```diff
@@ -352,11 +352,26 @@ function handleSend(params, context) {
         gasPrice: calculateGasPrice(params["bot-db"]["sliderValue"])
     };
 
-    try {
-        return web3.eth.sendTransaction(data);
-    } catch (err) {
-        return {error: err.message};
-    }
+
+    web3.eth.sendTransaction(data, function(error, hash) {
+        if (error) {
+            status.sendSignal("handler-data", {
+                status: "failed",
+                messageId: context["message-id"],
+                error: error
+            });
+        } else {
+            status.sendSignal("handler-data", {
+                status: "confirmed",
```

Comment:
Let's use "sent" here, "confirmed" will be used for confirmations from blockchain